### PR TITLE
Reformat ckarray.{C,h}

### PR DESCRIPTION
**Pull request by epmikida**
This was automated reformating with clang-format

### CODE REVIEW

**Comment by epmikida** [comment]
At Ronak's request, this moved reformatting from #3313 to it's own change.

File: src/ck-core/ckarray.h [hunk 1]
```diff
@@ -87,191 +88,242 @@ class CkVerboseListener : public CkArrayListener {
 /*@{*/
 
 /*********************** Proxy Support ************************/
-//Needed by CBase_ArrayElement
-class ArrayBase { /*empty*/ };
+// Needed by CBase_ArrayElement
+class ArrayBase
+{ /*empty*/
```
Comment (Wingpad) [inline src/ck-core/ckarray.h:93]: Would it be better to simply delete `/*empty*/` under the new formatting guidelines?

**Review by rbuch — APPROVED** [review]
We should consider doing something like this for pure reformatting changes, too: https://www.moxio.com/blog/43/ignoring-bulk-change-commits-with-git-blame.

File: src/ck-core/ckarray.h (hunk 1 above)
Comment (epmikida) [reply src/ck-core/ckarray.h]: Done

**Review by jszaday — DISMISSED** [review]

**Comment by epmikida** [comment]
> We should consider doing something like this for pure reformatting changes, too: https://www.moxio.com/blog/43/ignoring-bulk-change-commits-with-git-blame.

Yeah this seems like it would be a good idea

**Comment by epmikida** [comment]
I can't tell why this keeps failing. It's timing out occasionally on runs where Charm++ doesn't even finish initialization. I don't think it is specific to this PR, but it seems to be happening more and more frequently. Almost always in ZC tests, but this time it looks like it was in util.

**Review by rbuch — APPROVED** [review]

**Comment by epmikida** [comment]
> > We should consider doing something like this for pure reformatting changes, too: https://www.moxio.com/blog/43/ignoring-bulk-change-commits-with-git-blame.
> 
> Yeah this seems like it would be a good idea

I tried this out and it works. I'll merge this PR and then if we think its a good idea, we can set up a separate PR to add these revs to a blame file that ignores them.

**Review by jszaday — APPROVED** [review]

**Review by rbuch — APPROVED** [review]

**Pull request merged**